examples/Backtranslation_of_SQL_queries.py

```diff
@@ -24,17 +26,15 @@ def get_candidates(
     :param n: The number of completions to generate.
     :return: A list of completions.
     """
-    response = openai.Completion.create(
-        engine=engine,
-        prompt=prompt,
-        temperature=temperature,
-        max_tokens=150,
-        top_p=1,
-        frequency_penalty=0,
-        presence_penalty=0,
-        stop=stop,
-        n=n,
-    )
+    response = client.completions.create(engine=engine,
```

Comment:
Here. This code is not gonna work. completion.create doesn't have an engine parameter. 
completions API. soos will be deprecated. 
use client.chat.completions.create()

Reply:
the whole function should be upgraded to client.chat.completions.create(). Many notebooks still use this old function. so after a few months, I guess it's not gonna work.

Reply:
I see, this may have been an error in the migration script. I'll double check occurences of this. Thank you for the pointer @ARajgor!